app/jobs/create_impression_job.rb

```diff
@@ -1,7 +1,13 @@
 class CreateImpressionJob < ApplicationJob
   queue_as :impression
 
-  def perform(id, campaign_id, property_id, ad_template, ad_theme, ip_address, user_agent, displayed_at_string)
+  def perform(*args)
+    # Protection for impressions during deploy. Can remove after successful deploy
+    args = args.insert(5, "false") if args.length == 8
+    create_impression(*args)
+  end
+
+  def create_impression(id, campaign_id, property_id, ad_template, ad_theme, uplift, ip_address, user_agent, displayed_at_string)
```

Comment:
Might be safer to tack `uplift` to the end of the args and default it to `false`. Thoughts?

Reply:
oh.. good call!